test/rolodex/processors/swagger_test.exs

```diff
@@ -1,13 +1,20 @@
 defmodule Rolodex.Processors.SwaggerTest do
   use ExUnit.Case
 
-  alias Rolodex.{Route, Schema}
+  alias Rolodex.{Config, Route, Schema}
   alias Rolodex.Processors.Swagger
   alias Rolodex.Mocks.{User, NotFound}
 
   describe "#process/3" do
     test "Processes config, routes, and schemas into a serialized JSON blob" do
-      config = Rolodex.Config.new(description: "foo", title: "bar", version: "1")
+      config =
+        Config.new(
+          description: "foo",
+          title: "bar",
+          version: "1",
+          server_urls: ["https://api.dev.frame.io"]
```

Comment:
Such a nit, but lets not leave our dev api address in here. Can you move to example.com or something?